samples/grpc-server/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 spring.application.name=grpc-server
+grpc.server.reflection.enabled=false
```

Comment:
I think we actually need this to be true (and then we can remove the explicit `@Bean` definition in the app) so that the instructions in the README actually work as described.

Reply:
Updated.